docs/en/request-authorization-middleware.rst

```diff
@@ -78,3 +78,28 @@ AuthorizationMiddleware::
         $middlewareQueue->add(new AuthorizationMiddleware($this));
         $middlewareQueue->add(new RequestAuthorizationMiddleware());
     }
+
+Controller Usage
+================
+
+When having fallback routing activated, all asset based 404s as well as not existing controllers would also
+trigger the RequestAuthorizationMiddleware.
+In this case, it is possible to use this middleware only for your (App)controller::
+
+    public function initialize(): void
```

Comment:
Could we include a comment indicating which file this should go in?

Reply:
Its in the sentence before: "for your (App)controller"
But we can make it more clear, sure.

Reply:
@markstory Done.